doc/source/getting_started/install.rst

```diff
@@ -219,7 +219,7 @@ Dependencies
 ================================================================ ==========================
 Package                                                          Minimum supported version
 ================================================================ ==========================
-`setuptools <https://setuptools.readthedocs.io/en/latest/>`__    24.2.0
+`setuptools <https://setuptools.readthedocs.io/en/latest/>`__    38.6.0
```

Comment:
from https://medium.com/@daveshawley/safely-using-setup-cfg-for-metadata-1babbe54c108 

> If you want to use configuration in setup.cfg, then you should make life a little better on your users and include the pkg_resources.require call in your setup.py. It means fewer confused users creating issues for you and makes even educate your users about one of the dirtier corners of python packaging.

Reply:
Or maybe set a min setuptools version in pyproject.toml?